### PR TITLE
Enabled session expiry and track GC state by default

### PR DESCRIPTION
**Pull request by agarwal-navin**
Session expiry and trackGCState are disabled by default. They could however be enabled via feature flags / config provider settings to test that they work as expected. We have had them enabled for a while now and they have been working fine.
This change enables them by default.

[AB#3498](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/3498)

### CODE REVIEW

File: packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts [hunk 1]
```diff
@@ -642,79 +565,102 @@ describe("Garbage Collection configurations", () => {
...
 						gc.configs.shouldRunSweep,
-						testCase.expectedResult,
-						"shouldRunSweep not set as expected",
+						testCase.expectedShouldRunSweep,
+						`shouldRunSweep not set as expected`,
+					);
+				});
+			});
+		});
+		describe("sessionExpiryTimeoutMs and sweepTimeoutMs", () => {
```
Comment (markfields) [inline packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts:632]: I feel like these cases are covered already but no harm in laying it out this way, it makes the point clearly.

**Review by markfields — APPROVED** [review]
🎂

File: packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts (hunk 1 above)
Comment (agarwal-navin) [reply packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts:632]: okay. Do you think we should leave a feature flag to disable session expiry? With this change, there is no way to do that except for disabling GC.
I think its fine but wanted to get your opinion.

Comment (agarwal-navin) [reply packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts]: Added the setting back with default = session expiry enabled.

**Pull request merged**